material-overrides/home.html

```diff
@@ -13,16 +13,15 @@
   <div class="hero">
     <div class="hero-content">
       <h1>
-        Simplify deployments and<br />
-        management of AI workloads
+        Every AI response, verified
       </h1>
       <p class="hero-sub">
-        Explore how to use <a href="https://www.kluster.ai" class="hero-link" target="_blank">kluster.ai</a> to run Large Language Models on a distributed AI platform from providers all around the globe.
+        Deliver accurate, trustworthy results—every time. Integrate real-time validation into any AI workflow.
       </p>
 
       <div class="button-wrapper">
         <a class="framer-Bja7l framer-1v7ikbi framer-v-1v7ikbi framer-aotcnd" 
-           href="/get-started/start-building/setup/" 
+           href="/verify/reliability-check/verify-api/" 
```

Comment:
```suggestion
           href="/verify/reliability/verify-api/" 
```

oops this one changed too